docker-compose.yml

```diff
@@ -17,12 +17,12 @@ services:
     network_mode: bridge
     ports:
       - 5000:5000
-  
+
   uta:
     # Test:
-    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-    # 249909
-    image: biocommons/uta:uta_20180821
+    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
+    # 314227
```

Comment:
is it weird that this number changes?

Reply:
I think it's because it's getting newer transcripts that weren't there 3 years ago

Reply:
Was `20210129b` built on the same data as `20210129`?

Reply:
Ah, given the versioning I assumed it was trying to stick to the same data. Makes sense!

Reply:
Yes, `20210129b` and `20210129` were built on the same data.

Reply:
From my local uta:
![Screenshot 2023-09-11 at 5 54 43 PM](https://github.com/ga4gh/vrs-python/assets/42851808/97f4ac75-4396-42e9-b227-174e0c85f9df)